packages/stark-ui/src/modules/table/directives/table-row-content.directive.ts

```diff
@@ -0,0 +1,36 @@
+import { Directive, TemplateRef } from "@angular/core";
+
+/**
+ * This directive can be used inside <stark-table> if you want to customise the render of the table's cells
+ * A set of variables are available and therefore can be used to get the proper result :
+ * - rowData : is an array that contains information about a data row
+ * - cellRawValue : is the value of the cell unformatted
+ * - cellDisplayedValue : is the foramatted value of the cell
+ *
+ * @example
+ * <stark-table>
+ *     <ng-container [ngSwitch]="true" *starkTableRowContent="let rowData = rowData; let cellRawValue = rawValue; let cellDisplayedValue = displayedValue">
+ *          <!-- anycode you want to use to display table's cells -->
+ *
+ *          <!-- ie: if you want to add an icon under certain circomstances -->
+ *      	<div *ngSwitchCase="rowData.id === 1">
+ * 				<span style="color: blue">{{ cellDisplayedValue }}</span>
+ * 			</div>
+ *      	<div *ngSwitchCase="rowData.id === 2">
+ * 				<span style="color: red">{{ cellDisplayedValue }}</span>
+ * 			</div>
+ *      	<div *ngSwitchCase="cellRawValue > 23">
+ * 				<mat-icon class="mat-icon-rtl-mirror" svgIcon="thumb-up"></mat-icon> {{ cellDisplayedValue }}
+ * 			</div>
+ * 			<div *ngSwitchDefault>{{ cellDisplayedValue }}</div>
+ *     </ng-container>
```

Comment:
This code is not well formatted, could you please have a look ?
Also, I think it's a bit too complex for an example. Maybe you could remove the 
```
<div *ngSwitchCase="rowData.id === 2">
 * 				<span style="color: red">{{ cellDisplayedValue }}</span>
 * 			</div>
```
? 😊